ansible/roles/probe-services/templates/probe-services-nginx

```diff
@@ -141,5 +157,19 @@ server {
         return 200 "{{ probe_services_domain }}";
     }
 
+    # Haproxy on localhost dashboard - see roles/haproxy/README.adoc
+    # - /haproxy_stats
```

Comment:
Is it OK for this endpoint to be publicly visible or might it present security concerns?